gk/main.c

```diff
@@ -1951,6 +1951,11 @@ run_gk(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto out;
 	}
 
+	net_conf->gatekeeper_max_pkt_burst =
+		RTE_MAX(net_conf->gatekeeper_max_pkt_burst,
+			RTE_MAX(gk_conf->front_max_pkt_burst,
+				gk_conf->back_max_pkt_burst));
```

Comment:
Once `gatekeeper_max_pkt_burst` is renamed to `gatekeeper_total_pkt_burst`, the calculation above should become
```
net_conf->gatekeeper_total_pkt_burst = RTE_MAX(gk_conf->front_max_pkt_burst, gk_conf->back_max_pkt_burst) * gk_conf->num_lcores;
```
`RTE_MAX()` is still needed because a GK block will be either processing its queue at the front or at the back interface, but not both queues at the same time. `* gk_conf->num_lcores` accounts for each intstance.

This comment also applies to any functional block that accesses both interfaces (i.e. front and back) OR has multiple instances. That is, this comment also applies to LLS, CPS, and GT.

Reply:
Instead of `=` use `+=`.